deploy/crds/ocmagent.managed.openshift.io_ocmagents.yaml

```diff
@@ -34,10 +34,55 @@ spec:
           metadata:
             type: object
           spec:
-            description: OCMAgentSpec defines the desired state of OCMAgent
+            description: OcmAgentSpec defines the desired state of OcmAgent
```

Comment:
The actual field name is `ocmAgentConfig` but API docs say `OcmAgentSpec`. This might cause confusion to users via `oc explain ocmAgent.spec`. We could perhaps correct this for other fields as well in this API schema to avoid errors such as `error: field "OcmAgentSpec" does not exist`?

Reply:
this was generated automatically, I think it should be OcmAgentSpec? the OcmAgentCofnig is a sub-filed for the spec.

Reply:
Oh ok. I meant to convey that the API description could have the json field name (`ocmAgentConfig`) which the end user will have to configure rather than the internal Golang type which is (`OcmAgentConfig`). As an example:

```bash
$ oc explain upgrade.spec.upgradeAt
KIND:     UpgradeConfig
VERSION:  upgrade.managed.openshift.io/v1alpha1

FIELD:    upgradeAt <string>

DESCRIPTION:
     Specify the upgrade start time
$
$ oc explain upgrade.spec.UpgradeAt
error: field "UpgradeAt" does not exist
```
Currently, if users were to configure `OcmAgentConfig` in `.spec`, I think it would fail since such a field doesn't exist.

Reply:
Well, the correct field can as such be referred from the json field shown in `oc explain` so not a concern. All good 👍🏼